yang/ietf-l3vpn-ntw.yang

```diff
@@ -2314,6 +2314,14 @@ module ietf-l3vpn-ntw {
                         description
                           "Indicates the IP address of the peer.";
                       }
+                      leaf virtual-ip-address {
+                        type inet:ip-address;
+                        description
+                          "Virtual IPv4 addresses for a single VRRP group";
+                        reference
```

Comment:
No need to have an IPv4-specific reference as the type is inet:ip-address. I suggest to remove the reference statement.